### PR TITLE
Gputop documentation

### PR DESCRIPTION
**Pull request by yogesh-marathe**
Explains how to use gputop with celadon taget.

Signed-off-by: Yogesh Marathe <yogesh.marathe@intel.com>
Signed-off-by: Ravishankar Karkala Mallikarjunayya <ravishanakarx.karkala.mallikarjunayya@intel.com>

### CODE REVIEW

File: source/tutorials/profiling-gpu/profiling-gpu.rst [hunk 1]
```diff
@@ -0,0 +1,134 @@
+Profiling GPU
+########################################################################
+
+HW pre-requisites
+===========================
+Build machine/PC with Ubuntu 16.04
```
Comment (ttzeng) [inline source/tutorials/profiling-gpu/profiling-gpu.rst]: I think the rendering will look more organized if we put these prerequisites in a bulleted list.

File: source/tutorials/profiling-gpu/profiling-gpu.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+USB drive 8GB, ext4 formatted.
+
+USB type A connection from build machine to target.
```
Comment (ttzeng) [inline source/tutorials/profiling-gpu/profiling-gpu.rst]: Add comment that the cable is used only adb is enabled on the specific USB port.

File: source/tutorials/profiling-gpu/profiling-gpu.rst [hunk 3]
```diff
@@ -0,0 +1,135 @@
+Profiling GPU
+########################################################################
+
+HW pre-requisites
+===========================
+Build machine/PC with Ubuntu 16.04
+
+USB drive 8GB, ext4 formatted.
+
+USB Type-A to Type-A cable for build machine to target connection.
```
Comment (ttzeng) [inline source/tutorials/profiling-gpu/profiling-gpu.rst]: Add comment that this USB cable is used only adb is enabled on the specific USB port

**Comment by ttzeng** [comment]
@yogesh-marathe it looks good, but if you could revert the screenshot `gpu_top_command.png` to the low resolution one, that will be more readable to the readers.

**Comment by yogesh-marathe** [comment]
Done, updated the image to be more readable.

**Pull request merged**